cli/run.py

```diff
@@ -43,24 +44,42 @@ def kill_process_group_afterwards():
     help="Run an experiment on a scenario",
     context_settings=dict(ignore_unknown_options=True),
 )
+@click.option(
+    "--clean",
+    is_flag=True,
+    default=False,
+    help="Clean previously generated artifacts first",
+)
```

Comment:
@saulfield Is this necessary now? I thought it is no longer necessary to call clean.

Reply:
It's not necessary, but it is still an option.